plugins/rapid7_surface_command/plugin.spec.yaml

```
@@ -0,0 +1,73 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_surface_command
+title: Rapid7 Surface Command
+description: Surface Command gives you full visibilty over your assets and identies across multiple technology platforms.
+version: 1.0.0
+supported_versions: ["v1.0.790"]
+vendor: rapid7
+status: []
+tags: [caasm]
+
+hub_tags:
+  use_cases: [data_enrichment]
+  keywords: [caasm, rapid7, surface_command, cloud_enabled]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/example
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.rapid7.com
+
+cloud_ready: true
+enable_cache: false
+sdk:
+  type: slim
+  version: 5
+  user: nobody
+
+references:
+  - '[Identify Data Region](https://docs.rapid7.com/insight/navigate-the-insight-platform/#check-your-data-region)'
+  - '[Generate User API Key](https://docs.rapid7.com/insight/managing-platform-api-keys/#generating-a-user-key)'
+
+connection:
+  api_key:
+    title: API Key
+    description: User or Organization Key from the Insight Platform
+    type: credential_secret_key
+    example: a5zy0a6g-504e-46bz-84xx-1b3f5ci36l99
+    required: true
+  region:
+    title: Region
+    description: Region
+    type: string
+    default: United States
+    example: United States
+    required: true
+    enum:
+      - United States
+      - United States 2
+      - United States 3
+      - Europe
+      - Canada
+      - Australia
+      - Japan
+
+actions:
+  run_query:
+    title: Run Surface Command Query
+    description: Run and execute Surface Command Query
+    input:
+      query_id:
+        title: ID of Query to run
+        description: Query ID (UUID) to run from Surface Command
+        type: string
+        required: true
+        example: 12345678-1234-1234-1234-123456789012
+    output:
+      items:
+        title: Items
+        description: Array of Items
+        type: items
```

We might be able to change this to correspond to our base types "[]object" or "[]string" depending on what we expect items to be.
```suggestion
        type: "[]object"
```